src/ec/packages/ec-component-timeline/timeline.story.js

```diff
@@ -1,47 +1,69 @@
-/* eslint-disable no-param-reassign */
-
+/* eslint-disable no-param-reassign,prefer-destructuring */
 import { storiesOf } from '@storybook/html';
-import { withKnobs, number } from '@storybook/addon-knobs';
+import { withKnobs, number, text, select } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import { getExtraKnobs, buttonLabels } from '@ecl-twig/story-utils';
 import withCode from '@ecl-twig/storybook-addon-code';
 import iconPath from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+import he from 'he';
 
 import demoData from './demo/data';
 
 import timeline from './ecl-timeline.html.twig';
 import notes from './README.md';
 
+const prepareTimeline = data => {
+  const from = data.hide.from;
+  const to = data.hide.to;
+  let hiddenCount = 0;
+  if (to > 0) {
+    hiddenCount = to - from;
+  } else {
+    hiddenCount = data.items.length + to - from;
+  }
+  data.toggle_collapsed = select(
+    `data.toggle_collapsed`,
+    [`Show ${hiddenCount} more items`],
+    `Show ${hiddenCount} more items`,
+    buttonLabels.required
+  );
+  data.toggle_expanded = select(
+    `data.toggle_expanded`,
```

Comment:
same, can you put normal simple quotes here?

Reply:
done